vue-frontend/src/components/portfolio/DesignSection.vue

```diff
@@ -26,23 +26,21 @@
       <img
         v-if="response[0].responsiveImages"
         :src="response[0].backgroundImage[3]"
-        :srcset="`${response[0].backgroundImage[0]} 400w, ${response[0].backgroundImage[1]} 800w, ${response[0].backgroundImage[2]} 1200w, ${response[0].backgroundImage[3]} 1600w`"
-        sizes="(min-width: 992px) 45vw, 100vw"
-        class="tw-w-full tw-h-full tw-object-cover"
-        :alt="response.alt"
-      />
-      <img
-        v-else
-        :src="response[0].backgroundImage"
+        :srcset="`${response[0].backgroundImage[0]} 400w, ${response[0].backgroundImage[1]} 800w, ${response[0].backgroundImage[2]} 1400w, ${response[0].backgroundImage[3]} 2400w`"
+        sizes="
+        (min-width: 2420px) 2200px, 
+    (min-width: 720px) calc(94.76vw - 274px), 
+    (min-width: 520px) calc(100vw - 96px), 
+    calc(100vw - 32px)"
         class="tw-w-full tw-h-full tw-object-cover"
         :alt="response.alt"
       />
     </aspect-ratio>
-    <img
-      v-if="response[0].gif"
-      :src="response[0].gif"
-      class="tw-absolute tw-inset-0 tw-left-auto tw-h-2/4 tw-w-2/6 tw-object-cover cycling-animation"
+    <LottieAnimation
```

Comment:
Have you checked on real mobile devices? I think they not working properly on mobiles

Reply:
its working well in all devices even in IOS also.